downstream/modules/platform/proc-create-postresql-secret.adoc

```diff
@@ -0,0 +1,30 @@
+[id="create-postresql-secret_{context}"]
+
+= Creating a postgresql configuration secret
+
+[role=_abstract]
+
+For migration to be successful, you must provide access to the database for your existing deployment.
+
+.Procedure
+
+. Create a yaml file for your postgresql configuration secret:
++
+-----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <resourcename>-old-postgres-configuration
+  namespace: <target namespace>
+stringData:
+  host: <external ip or url resolvable by the cluster>
```

Comment:
Can we add quotes around each of the values under "stringData", like how we did for the external db docs?  This may help customers avoid issues when not adding quotes around the port value.  

![image](https://user-images.githubusercontent.com/11698892/195626230-03a6afa5-056b-4857-887a-49476aa5a6dc.png)

Reply:
Changes made.